docs/index.adoc

```diff
@@ -592,6 +592,115 @@ public class SecuredServiceEndpoint {
 
 For more details see this guide: https://openliberty.io/guides/microprofile-jwt.html[Securing microservices with JWTs]
 
+== Messaging
+
+Applications can pass messages among themselves with the Java Message Service (JMS) APIs. Liberty can act as a server of JMS destinations (queues and topics,) or it can connect to another server to access those destinations--either another Liberty server or IBM MQ.
```

Comment:
Could you add an extra sentence/paragraph explaining what queues and topics are and when to use one vs. the other? AFAIK a Queue is when each message is processed exactly once, and a Topic is more like a subscription model where zero-to-many processes could read each message.

Reply:
Also, are the only other supported destination servers Liberty or IBM MQ? For example, does Kafka work?

Reply:
I honestly don't know about others. They _should_ work generically if there exists an appropriate resource adapter similar to MQ does it but I didn't want to speak when I had no experience trying it out. I can throw in a sentence about other JMS-compatible messaging platforms.

Reply:
That's fine if you're not certain about others. If you're not certain we don't need to name them explicitly, but we should at least allude to the possibility of other compatible messaging servers besides the the one listed (i.e. saying "a or b" vs saying "a, b, etc")